app/controllers/sessions_controller.rb

```diff
@@ -21,7 +27,6 @@ def destroy
 
   def find_user_base_session
     @user = User.find_by email: params.dig(:session, :email)&.downcase
-
     return if @user.nil?
```

Comment:
code này đúng ko, t chém chừ 
```suggestion
    return if @user
```